app/modules/users/components/user_profile.svelte

```diff
@@ -0,0 +1,143 @@
+<script>
+  import { i18n } from '#user/lib/i18n'
+  import { icon } from '#lib/utils'
+  import { imgSrc } from '#lib/handlebars_helpers/images'
+  import { userContent } from '#lib/handlebars_helpers/user_content'
+  import Flash from '#lib/components/flash.svelte'
+  import { screen } from '#lib/components/stores/screen'
+  import UserProfileButtons from '#users/components/user_profile_buttons.svelte'
+
+  export let user
+
+  const { username, bio, picture, inventoryLength, shelvesCount } = user
+
+  let flash
+</script>
+
+<div class="user-profile">
+  <div class="user-card">
+    <div class="avatar-wrapper">
+      <img class="avatar" src={imgSrc(picture, 150, 150)} alt="{username} avatar">
+    </div>
+    <div class="info">
+      <h2 class="username respect-case">{username}</h2>
+      <ul class="data">
+        {#if inventoryLength != null}
+          <li class="inventoryLength">
+            <span>{@html icon('book')}{i18n('books')}</span>
+            <span class="count">{inventoryLength}</span>
+          </li>
+        {/if}
+        {#if shelvesCount != null}
+          <li class="showShelvesList shelvesLength">
+            <span>{@html icon('server')}{i18n('shelves')}</span>
+            <span class="count">{shelvesCount}</span>
+          </li>
+        {/if}
+      </ul>
+      {#if $screen.isLargerThan('$smaller-screen')}
+        {#if bio}
+          <p class="bio-wrapper">{@html userContent(bio)}</p>
+        {/if}
+      {/if}
+    </div>
+  </div>
+
+  {#if $screen.isSmallerThan('$smaller-screen')}
+    {#if bio}
+      <p class="bio-wrapper">{@html userContent(bio)}</p>
+    {/if}
+  {/if}
+
+  <UserProfileButtons {user} bind:flash />
+</div>
+
+<Flash state={flash} />
+
+<style lang="scss">
+  @import '#general/scss/utils';
+  .user-profile{
+    background-color: #eee;
+    @include display-flex(row);
+    margin: 0.5em 0;
+  }
+  .user-card{
+    flex: 1;
+    @include display-flex(row);
+  }
+  .avatar{
+    background-color: #eee;
+  }
+  .info{
+    position: relative;
+  }
+  .username{
+    @include sans-serif;
+    margin: 0;
+  }
+  .data{
+    @include display-flex(row, flex-start);
+    color: #666;
+    margin-bottom: 0.5em;
+    li{
+      margin-right: 1em;
+    }
+  }
+  .inventoryLength, .showShelvesList{
+    .count{
+      padding-left: 0.5em;
+    }
+  }
+  .bio-wrapper{
+    @include radius;
+    margin-bottom: 2em;
+    max-width: 50em;
+    overflow: auto;
+  }
+
+  /*Large screens*/
+  @media screen and (min-width: $smaller-screen) {
+    .avatar-wrapper{
+      flex: 0 0 auto;
+    }
+    .info{
+      flex: 1 0 0;
+      padding: 0.8em 1em;
```

Comment:
On large but not so large screen (~650) the space below the profile is quite high. I would be for reducing it a bit 
```suggestion
      padding: 0.8em 1em 0 1em
```

Reply:
done in fc3aeba5b